src/stubs/gitattributes.txt

```diff
@@ -4,6 +4,7 @@
 /.gitattributes export-ignore
 /.gitignore export-ignore
 /.travis.yml export-ignore
+/.php_cs export-ignore
```

Comment:
I guess this needs to be refactored towards using the exportIgnores array of the Construct class anyways, but will keep that in mind when doing so ;D